utils/permissions.py

```diff
@@ -57,7 +57,8 @@ async def pred(ctx: discord.Interaction):
 
 async def check_priv(
     ctx: discord.Interaction,
-    member: discord.Member
+    member: discord.Member,
+    bool: bool=False, # Option To Return just a Boolean
```

Comment:
Consider not potentially shadowing builtins (`bool`).

Also checks shouldn't really send messages, instead raise a custom error and handle it in an error handler.